crates/tools/js/benchmark/index.js

```diff
@@ -89,6 +89,14 @@ async function verify(benchmarkResultPath) {
   ));
 
   for (let scenarioName in snapshotResult) {
+    // Snapshot testing is unreliable for these scenarios
+    if (
+      scenarioName.includes("openzeppelin") ||
```

Comment:
Could this be related to https://github.com/NomicFoundation/hardhat/issues/5036#issuecomment-2023343177?

Reply:
Just an FYI, when I enabled logging for a repo - that suffers from the described problem - and compared the generated logs, the order of calls wasn't deterministic. This was a sign to me that tests were being executed in parallel.

This might be a relatively straightforward way of testing the hypothesis for other repos.